docs/Getting-Started-with-Balance-Ball.md

```diff
@@ -272,6 +272,7 @@ To summarize, go to your command line, enter the `ml-agents` directory and type:
 ```python
 python3 python/learn.py <env_file_path> --run-id=<run-identifier> --train 
 ```
+**Note**: If you're using Anaconda, don't forget to activate the ml-agents environment first.
```

Comment:
What does "activate the ml-agents environment" mean? This would be the only place in the Docs that we mention that.